install/linux/dangerzone.spec

```diff
@@ -78,9 +78,6 @@ Requires:       libreoffice
 Requires:       podman
 %endif
 
-# FIXME: Is it autodetected?
-Requires:       python3-PyMuPDF
```

Comment:
That's great actually, because it showcases an issue with our PyMuPDF dependency: we have the wrong bounds. We should relax our bounds in `pyproject.toml`, so that the PyMuPDF version in Fedora 38 can be installed.

Reply:
Bit of a different issue, that's because I should use `python3-fitz` instead of `python3-pymupdf` here:

https://github.com/freedomofpress/dangerzone/pull/748/files#diff-aa574ae0e7c34331f294e51e54f5080d91890de1618c6c651402fd8881ae58e6

Again, nice catch. We should see all these issues light up once we have a proper CI. I'm working on that